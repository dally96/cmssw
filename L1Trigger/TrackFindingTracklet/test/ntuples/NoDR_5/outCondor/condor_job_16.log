000 (802068.016.000) 02/16 20:30:28 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_8027_6ecd>
...
040 (802068.016.000) 02/16 20:31:47 Started transferring input files
	Transferring to host: <188.184.165.142:9618?addrs=188.184.165.142-9618+[2001-1458-301-51--100-89]-9618&alias=b7s14p4818.cern.ch&noUDP&sock=slot1_3_3869_cf76_18860>
...
040 (802068.016.000) 02/16 20:31:47 Finished transferring input files
...
001 (802068.016.000) 02/16 20:31:48 Job executing on host: <188.184.165.142:9618?addrs=188.184.165.142-9618+[2001-1458-301-51--100-89]-9618&alias=b7s14p4818.cern.ch&noUDP&sock=startd_3109_1808>
...
006 (802068.016.000) 02/16 20:31:57 Image size of job updated: 47344
	33  -  MemoryUsage of job (MB)
	32832  -  ResidentSetSize of job (KB)
...
006 (802068.016.000) 02/16 20:33:37 Image size of job updated: 2867016
	33  -  MemoryUsage of job (MB)
	32844  -  ResidentSetSize of job (KB)
...
040 (802068.016.000) 02/16 20:33:37 Started transferring output files
...
040 (802068.016.000) 02/16 20:33:37 Finished transferring output files
...
005 (802068.016.000) 02/16 20:33:37 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:28, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:28, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	25002  -  Run Bytes Sent By Job
	25479  -  Run Bytes Received By Job
	25002  -  Total Bytes Sent By Job
	25479  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       63       17   2431966 
	   Memory (MB)          :       33     2000      2000 

	Job terminated of its own accord at 2022-02-16T19:33:37Z.
...
