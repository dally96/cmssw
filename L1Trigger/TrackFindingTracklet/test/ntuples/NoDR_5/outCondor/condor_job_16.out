/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test /pool/condor/dir_286224 
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 07:42:37
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 07:42:36
uri       : voms-cms-auth.app.cern.ch:443

subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 07:42:37
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 07:42:36
uri       : voms-cms-auth.app.cern.ch:443

cmsRun /afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/customise_L1TrackNtupleMaker.py inputFiles_clear inputFiles_load=/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/ntuples/NoDR_5/pFiles_16.txt outputFile=outputL1Ntuple_16.root print
Singletons:
  filePrepend        : 
                     - String to prepend location of all files
  maxEvents          : -1
                     - Number of events to process (-1 for all)
  outputFile         : outputL1Ntuple_16.root
                     - Name of output file (if needed)
  secondaryOutputFile: 
                     - Name of second output file (if needed)
  section            : 0
                     - This section (from 1..totalSections inclusive)
  tag                : 
                     - tag to add to output filename
  totalSections      : 0
                     - Total number of sections
Lists:
  inputFiles         : ['/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/a9ca7db6-3d9d-4592-8a95-00d10859df02.root' ]
                     - Files to process
  secondaryInputFiles: []
                     - Second group of files to process (if needed)

using geometry D76 (tilted)
Wed Feb 16 20:33:19 CET 2022
Thread 7 (Thread 0x2b246275f700 (LWP 264)):
#0  0x00002b24093e4b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b24093e4bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b24093e4c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b240f7bf816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b240f7bf8c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b246255e700 (LWP 263)):
#0  0x00002b24093e4b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b24093e4bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b24093e4c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b240f7bf816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b240f7bf8c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b246235d700 (LWP 262)):
#0  0x00002b24093e4b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b24093e4bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b24093e4c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b240f7bf816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b240f7bf8c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b246215c700 (LWP 261)):
#0  0x00002b24093e5e9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b240e75f6ff in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b240f761b0a in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#3  0x00002b240f761c69 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b2461f5b700 (LWP 260)):
#0  0x00002b24096f30e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b240e759ed2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b240e75678d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#3  0x00002b240e75ef68 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#4  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b2427f04700 (LWP 253)):
#0  0x00002b24093e61d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b240fabd387 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b240fabe01a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b2408fafaf0 in std::execute_native_thread_routine (__p=0x2b24270d9800) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b24093deea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b24096f2b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b240b76da40 (LWP 235)):
#0  0x00002b24096e7ddd in poll () from /lib64/libc.so.6
#1  0x00002b240fabd7b7 in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b240fabe0ec in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b240fac159b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b240962a387 in raise () from /lib64/libc.so.6
#6  0x00002b240962ba78 in abort () from /lib64/libc.so.6
#7  0x00002b2408f7a683 in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00002b2408f860a6 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00002b2408f851a9 in __cxa_call_terminate (ue_header=ue_header@entry=0x2b2471b43ae0) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00002b2408f85ad4 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386944, ue_header=0x2b2471b43ae0, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:677
#11 0x00002b24093cd7b3 in _Unwind_RaiseException_Phase2 (exc=0x2b2471b43ae0, context=0x7ffe653fa600, frames_p=0x7ffe653fa508) at ../../../libgcc/unwind.inc:64
#12 0x00002b24093ce016 in _Unwind_Resume (exc=0x2b2471b43ae0) at ../../../libgcc/unwind.inc:241
#13 0x00002b2407733b1c in TFile::Close(char const*) [clone .cold] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libRIO.so
#14 0x00002b240e9978ee in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#15 0x00002b240e997919 in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#16 0x000000000040f4d8 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() ()
#17 0x00002b2475459b6c in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#18 0x00002b2475459daf in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#19 0x00002b245e8afc03 in std::_Sp_counted_deleter<edm::one::OutputModuleBase*, std::default_delete<edm::one::OutputModuleBase>, std::allocator<void>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#20 0x00002b245e8b0132 in std::_Sp_counted_ptr_inplace<edm::maker::ModuleHolderT<edm::one::OutputModuleBase>, std::allocator<edm::maker::ModuleHolderT<edm::one::OutputModuleBase> >, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#21 0x00002b2406e39d7c in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#22 0x00002b2406e39d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#23 0x00002b2406e39d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#24 0x00002b2406e39dd6 in std::_Sp_counted_ptr<edm::ModuleRegistry*, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#25 0x00002b2406d50598 in std::default_delete<edm::Schedule>::operator()(edm::Schedule*) const [clone .isra.0] [clone .part.0] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#26 0x00002b2406d57be2 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#27 0x000000000040b851 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#28 0x000000000040b649 in main ()
